ts/components/session/conversation/SessionRecording.tsx

```diff
@@ -113,28 +139,37 @@ class SessionRecordingInner extends React.Component<Props, State> {
           onMouseEnter={this.handleHoverActions}
```

Comment:
do we still need the handleHoverActions at all?

Reply:
and the handleUnhoverActions ?

Reply:
I think it's unused actually so I'll remove it.